app/Models/Emulator.php

```diff
@@ -161,7 +161,10 @@ public function getHasOfficialSupportAttribute(): bool
         foreach ($this->userAgents as $userAgent) {
             // Case 1: Has minimum_hardcore_version. This implies there's at least one
             // version of the emulator out there which we fully support.
-            if ($userAgent->minimum_hardcore_version || $userAgent->minimum_allowed_version) {
+            if (
+                ($userAgent->minimum_hardcore_version && $userAgent->minimum_hardcore_version < 99999)
+                || ($userAgent->minimum_allowed_version && $userAgent->minimum_allowed_version < 99999)
+            ) {
```

Comment:
The `->active` flag is meant to drive this behavior. There's a lot of stuff not marked as active because we didn't want it showing up on the Downloads page. Since we do want it showing up on the new Downloads page, shouldn't this just look at `->active` and we can update `->active` after switching over?

Reply:
Yep - fixed in latest.